bftengine/src/bftengine/BFTEngine.cpp

```diff
@@ -116,9 +116,7 @@ IReplica::IReplicaPtr IReplica::createNewReplica(const ReplicaConfig &replicaCon
                                                  IStateTransfer *stateTransfer,
                                                  bft::communication::ICommunication *communication,
                                                  MetadataStorage *metadataStorage,
-                                                 bool &erasedMetadata,
```

Comment:
Thanks for removing that. It was confusing, especially having the `erasedMetaData` local variable.